drivers/unix/file_access_unix.cpp

```diff
@@ -230,7 +230,7 @@ uint64_t FileAccessUnix::_get_modified_time(const String &p_file) {
 	if (success) {
 		return flags.st_mtime;
 	} else {
-		print_line("ERROR IN: " + p_file);
+		print_line("Cannot read (it might be a symbolic link): " + p_file);
```

Comment:
There might be other reasons why it would fail, I wouldn't make the message so specific.

Reply:
Got your point